x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/insights/endpoint_insights_results.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import {
+  EuiButtonIcon,
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import { ENDPOINT_INSIGHTS } from '../../../translations';
+
+interface EndpointInsightsResultsProps {
+  results: boolean;
+}
+
+const CustomEuiCallOut = styled(EuiCallOut)`
```

Comment:
I think we're moving away from `styled` in favor of `emotion` - consider changing :)